vue-app/src/views/RecipientRegistry.vue

```diff
@@ -25,10 +25,10 @@
             >
               <td>
                 <div class="project-name">
-                  <links :to="request.metadata.thumbnailImageUrl">
+                  <links :to="request.metadata.imageUrl">
```

Comment:
Should this be changed? I believe we want to keep it as `thumbnailImageUrl`.

Reply:
Yes, I've added it into the `getRequests` api function under `api/recipient-registry-optimistic.ts` because it wasn't there so it was always undefined.